scripts/import/import_gene_disease.py

```diff
@@ -1,18 +1,95 @@
 #!/usr/bin/env python3
 
-import os
+import sys
 import re
 import argparse
 import datetime
 import MySQLdb
 import xml.etree.ElementTree as ET
-
-debug = 0
+import warnings
+import csv
+import configparser
+from pathlib import Path
+from os import listdir
 
 """
-    Retrieve OMIM gene disease data from Ensembl core db
+    Script      : import_gene_disease.py
+
+    Description : Imports/updates gene-disease associations from OMIM (Ensembl) and Mondo
+                  The script can be run in two modes:
+                    - import (default) : Runs a full import of the data. This mode should only be run on a newly created db.
```

Comment:
Suggesting that update is made the default, as the import requires to be run on a newly created db which would be on very rare occasion but the update will be for every release cycle